algosdk/transaction.py

```diff
@@ -440,17 +440,17 @@ class KeyregTxn(Transaction):
         note (bytes)
         genesis_id (str)
         genesis_hash (str)
-        group(bytes)
-        votepk (str)
-        selkey (str)
+        group (bytes)
+        votekey (byte[32])
+        selkey (byte[32])
         votefst (int)
         votelst (int)
         votekd (int)
         type (str)
         lease (byte[32])
         rekey_to (str)
         nonpart (bool)
-        sprfkey (str)
+        sprfkey (byte[64])
     """
 
     def __init__(
```

Comment:
minor: it'd be nice to annotate the args with `Union[bytes, bytearray, str]` so it's clearer what types are accepted (though I realize this class does not do any type annotating)